tests/src/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -306,4 +306,53 @@ class NodeJsActionContainerTests extends FlatSpec with Matchers {
         filtered(out).trim shouldBe empty
         filtered(err).trim shouldBe empty
     }
+
+    it should "error when requiring a non-existent package" in {
+        // NPM package names cannot start with a dot, and so there is no danger
+        // of the package below ever being valid.
+        // https://docs.npmjs.com/files/package.json
+        val (out, err) = withNodeJsContainer { c =>
+            val code = """
+                | function main(args) {
+                |     require('.mildlyinvalidnameofanonexistentpackage');
```

Comment:
@danielfederschmidt I think your point is still technically valid, and the change was easy enough to make, so I went ahead and did it 😉